src/hooks/usePlaceOrder.ts

```diff
@@ -147,8 +147,8 @@ export const usePlaceOrder = (): Result => {
           buyTokens.push(order.buyToken)
           sellTokens.push(order.sellToken)
 
-          // if not set, order is valid from placement
-          validFroms.push(order.validFrom || currentBatchId)
+          // if not set, order is valid from placement + wait period
+          validFroms.push(order.validFrom || currentBatchId + BATCHES_TO_WAIT)
```

Comment:
Wanna use the new fancy null coalescence here?
```suggestion
          validFroms.push(order.validFrom ?? currentBatchId + BATCHES_TO_WAIT)
```

It would allow for `validFrom` to be 0, not that it will ever be 0

Reply:
:+1:

Reply:
Well, why not?